pcbuilder_crawlers/template.py

```diff
@@ -1,10 +1,12 @@
 import json
-import os
+
 
 class TemplateInterface(object):
-    def get_template(key):
-            template = None
-            return template
+    def get_template(self, category):
+        path = 'templates/{category}.json'.format(category=category)
+        with open(path) as data:
```

Comment:
http://stackoverflow.com/questions/10971033/backporting-python-3-openencoding-utf-8-to-python-2

`with io.open(path, 'rb') as data:`

This should prevent that encoding issue.